### PR TITLE
add popover info example to styleguide

### PR DESCRIPTION
**Pull request by anthonyettinger**
add popover info example to styleguide

@seanbrookes

### CODE REVIEW

File: client/www/scripts/modules/common/common.directives.js [hunk 1]
```diff
@@ -242,6 +243,55 @@ Common.directive('slPopoverMenu', [
     };
   }]);
 
+Common.directive('slPopoverInfo', [
```
Comment (seanbrookes) [inline client/www/scripts/modules/common/common.directives.js]: can you add the module to the directive name?

makes it easier to find the directive when troubleshooting

**Comment by seanbrookes** [comment]
just one comment other than that LGTM

File: client/www/scripts/modules/common/common.directives.js (hunk 1 above)
Comment (anthonyettinger) [inline client/www/scripts/modules/common/common.directives.js]: I'm leaving off `common` namespace. for re-usable UI widgets.

**Comment by seanbrookes** [comment]
LGTM

**Pull request merged**